### ISSUE TITLE
[HOLD on #7925] Unable to scroll up occasionally

### ISSUE THREAD

**Issue by mvtglobally** [Expensify/App#8150]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Using trackpad scroll up in NewDot

## Expected Result:
It would continue to scroll til I got to the beginning of chat history

## Actual Result:
It stopped scrolling up, even though there were more chats.

## Workaround:
unknown 

## Platform:

Where is this issue occurring?

- Web - Firefox

**Version Number:** 1.1.42-5
**Reproducible in staging?:** Y 
**Reproducible in production?:** Y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation 
<img width="1231" alt="image - 2022-03-15T003151 017" src="https://user-images.githubusercontent.com/43995119/158306950-db178230-b3f9-44d1-9137-ad1a9fc73590.png">

https://user-images.githubusercontent.com/43995119/158306952-39674e61-c17e-44db-b306-c51cf1f91e2d.mp4

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @mallenexpensify 
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1647282942862949?thread_ts=1647276844.142609&cid=C01GTK53T8Q

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @nkuoch (`Engineering`), see https://stackoverflow.com/c/expensify/questions/4319 for more details.

**Comment by MelvinBot** [comment]
Triggered auto assignment to @JmillsExpensify (`External`), see https://stackoverflow.com/c/expensify/questions/8582 for more details.

**Comment by JmillsExpensify** [comment]
Issue looks good. Upwork job created here: https://www.upwork.com/jobs/~01de4e7dfd94d1b5ab

**Comment by MelvinBot** [comment]
📣 @rushatgabhane You have been assigned to this job by @melvin-bot[bot]!
Please apply to this job in Upwork and leave a comment on the Github issue letting us know when we can expect a PR to be ready for review 🧑‍💻
Keep in mind: [Code of Conduct](https://github.com/Expensify/App/blob/main/CODE_OF_CONDUCT.md) | [Contributing](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md) 📖

**Comment by mountiny** [comment]
I am not sure about this one at least based on the video provided and the current description. The video shows that we basically do not load the old messages which are paginated fast enough so that when you scroll up fast it stops as the App has not fetched ht messages yet.

If that would be a problem, I am not sure if there is much to do here, this is common in other chat apps and it will depend on the quality of the internet connection. 

A solution would be to start fetching the old messages sooner, however, that would increase the load of potentially unnecessary requests.

I think we should do nothing here and if the `jumping` problem with not smooth scrolling has some reproducible steps, we should create a new GH with a video of it and steps to reproduce specifically for that problem. What do you think @JmillsExpensify @mallenexpensify

**Comment by mallenexpensify** [comment]
I'm fine with 'do nothing' for now but... the issue isn't that it take a while to load, it's that there are times when you can't scroll up, even after waiting for the text to load.  It freezes and you have to scroll up/down in order to get it to unfreeze then scroll up more.  Which makes me think it's not just a loading or pagination issue

**Comment by mountiny** [comment]
> Which makes me think it's not just a loading or pagination issue

Interesting, not sure if I have reproduced that. I guess this would then go to `Needs Reproduction` so we can clearly find the problem once knowing how to achieve it.

Would you want to update this issue or create a new one with specifics for the problem you mentioned?

**Comment by mallenexpensify** [comment]
I'm able to reproduce it on web - Chrome.  Find a chat you know has a ton of content then try to keep scrolling up, like you're loooking for a chat from 6 months back.  Eventually it might just stop/freeze.  (I was unable to reproduce on Firefox but ran into another issue or two).  If you think we should put this on hold til we have bidirectional scrolling, that's fine too

**Comment by mountiny** [comment]
Alright, what I was able to reproduce is to wildly scroll back and at some point, it is not loading new messages. I think this could be related to some condition that triggers the fetching of the new messages.

I think it is a good idea to put this on hold while the bidirectional scrolling is implemented as that could introduce new behaviour!

**Comment by mallenexpensify** [comment]
Here's the bidirectional issue link, added to title too https://github.com/Expensify/App/issues/7925

**Comment by JmillsExpensify** [comment]
Just seeing this, but I've experienced this issue before as well. If you are scrolling through your history, and at a certain point, you just get stuck. So I do think this is an issue, though I agree it's not urgent. I'd actually recommend though that we just close it for now if we'd like to wait on bi-directional scrolling first. Feel free to re-open if you disagree.
